src/indexing_pipeline.py

```
@@ -831,6 +831,43 @@ async def load_existing():
+                        """,
+                            result.document_id,
+                            result.document_title,
+                            str(document_path),
+                            result.hierarchy_depth,
+                            len(chunks),
+                            json.dumps(doc_metadata)
+                        )
+                        logger.info(f"Registered document in metadata.documents: {result.document_id}")
+                    finally:
+                        await conn.close()
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)